waffle/templatetags/waffle_tags.py

```diff
@@ -81,3 +87,32 @@ def render(self, context):
 @register.tag
 def wafflejs(parser, token):
     return InlineWaffleJSNode()
+
+
+@register.filter(name="switch_is_active")  # type: ignore[misc]
+def switch_is_active(switch_name: str) -> bool:
+    """
+    This template filter works like the switch tag, but you can use it within
+    if statement conditions in Django templates. Example: `{% if "switch_name"|switch_is_active %}`.
+    """
+    return waffle_switch_is_active(switch_name)
+
+
+@register.simple_tag  # type: ignore[misc]
```

Comment:
question: should this have a `name` similar to `switch_is_active` and `sample_is_active`?